projects/minesweeper/src/main/java/org/batfish/minesweeper/question/searchroutepolicies/SearchRoutePoliciesQuestion.java

```diff
@@ -21,12 +21,10 @@
 
   private static final BgpRouteConstraints DEFAULT_ROUTE_CONSTRAINTS =
       BgpRouteConstraints.builder().build();
-  private static final String DEFAULT_NODES = "";
-  private static final String DEFAULT_POLICIES = "";
   private static final Action DEFAULT_ACTION = Action.PERMIT;
 
-  @Nonnull private final String _nodes;
-  @Nonnull private final String _policies;
+  @Nullable private final String _nodes;
+  @Nullable private final String _policies;
```

Comment:
I don't understand the goal -- what is this allowing that wasn't allowed before?  It seems like these things were already optional on the Client side, but by allowing null internally you're just introducing more possibility for null pointer exceptions.